src/dotnet/CommandFactory/LocalizableStrings.resx

```diff
@@ -220,4 +220,7 @@
   <data name="MSBuildArgs" xml:space="preserve">
     <value>MSBuild arguments: {0}</value>
   </data>
+  <data name="NeedRunToolRestore" xml:space="preserve">
+    <value>Please run \"dotnet tool restore\" to make command \"{0}\" available.</value>
```

Comment:
@KathleenDollard the only user facing string in this PR

Reply:
And I will make a separate discussion on if we want implicit restore

Reply:
I don't think we need the quote escaping in this context.  Additionally, let's not use the word "please" as we still need to scrub our existing uses.

How about `Run "dotnet tool restore" to make the "{0}" command available.`?

Reply:
Fixed as suggested